ml/Config.cc

```diff
@@ -84,10 +84,17 @@ void Config::readMLConfig(void) {
 
     Cfg.EnableAnomalyDetection = EnableAnomalyDetection;
 
+ #if 1
     Cfg.MaxTrainSamples = MaxTrainSamples;
     Cfg.MinTrainSamples = MinTrainSamples;
     Cfg.TrainEvery = TrainEvery;
-    Cfg.NumModelsToUse = NumModelsToUse;
+    Cfg.NumModelsToUse = 1;
```

Comment:
@vkalintiris just checking this is not hard coding `NumModelsToUse` to 1 or anything?

Reply:
cool - i will drop and recreate the test nodes when ready